.gitignore

```diff
@@ -40,3 +40,6 @@ config/customization/*
 config/transifex.netrc
 !config/provider/.gitkeep
 !config/customization/.gitkeep
+
+# ignore IDE configuration 
+.idea/
```

Comment:
It's for RubyMine.
True, we actual did add that to our global gitignore, so it doesn't really need to be here. 
(Now we just have to find out how to remove it. :) )